### PR TITLE
Header and headline sections of portfolio

### PR DESCRIPTION
**Pull request by Solo7991**
The two sections of the portfolio
Linters such as lighthouse, webhint, and stylelint

### CODE REVIEW

**Review by Danieldotcomcoder — COMMENTED** [review]
Hi @Solo7991, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive Readme file 😃
Please use this  [Template]( https://github.com/microverseinc/readme-template/blob/master/README.md) to add a readme file to your project ✔👍


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏👍✔

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by JuliCarracedo — COMMENTED** [review]
Hi @Solo7991 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not using any of the provided [Figma designs](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Even though your design is really interesting, you ar ealso being tested on your ability to translate designs into technical solutions so you must use one of the provided designs. I'd recommend Template 5 since it's the closest to your aspirations.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            <img src="icons/git.svg" alt="icon">
+          </div>
+          <div>
+            <img src="icons/chi.svg" alt="icon">
+          </div>
+        </div>
+        <!-- icon end -->
+
+      </div>
+      <!-- headline end -->
```
Comment (uche-inyama) [inline index.html:87]: Kindly wrap this section with a **section** tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          <div>
+            <img src="icons/space.svg" alt="icon">
+          </div>
+          <div>
+            <img src="icons/git.svg" alt="icon">
+          </div>
+          <div>
+            <img src="icons/chi.svg" alt="icon">
+          </div>
+        </div>
```
Comment (uche-inyama) [inline index.html:83]: List the icons with a ul and li tags. Also, wrap the image with an **a** tag. As each icon should navigate to webpage.

## :slightly_smiling_face:

**Review by uche-inyama — CHANGES_REQUESTED** [review]
### Hi @Solo7991 :slightly_smiling_face: 

Good job so far, and I must commend your efforts.

There are some issues that you still need to work on, you are almost there!




## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @Solo7991 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**